src/rgw/rgw_op.cc

```diff
@@ -3718,6 +3732,9 @@ static CompressorRef get_compressor_plugin(const req_state *s,
 
 void RGWPutObj::execute(optional_yield y)
 {
+  this->trace->set_tag("object_name", this->s->object->get_name());
```

Comment:
why not setting the tag on the new span?

Reply:
> why not setting the tag on the new span?

I thought that the object name is more related to the first span.

Reply:
anything on the first span should be "op agnostic", so you can set it in the `init()` function
i think that object name may not apply to all ops, so its probably better in the child span